scripts/primary-map-features.js

```diff
@@ -13,20 +13,42 @@ if (!argv.output) {
   console.log('  OPTIONS');
   console.log('    --output counts.csv');
   console.log('');
-  return;
+  throw new Error('USAGE: error');
```

Comment:
@bkowshik I don't know, but putting a `return` without a function is a syntactical error I believe.